backend/libexecution/serialization_converters.ml

```diff
@@ -31,6 +31,8 @@ let parseString str :
     try
       (* Use strings here as js_of_ocaml is not able to deal with int literals
        * outside the 31-bit range *)
+      (* Edit: We no longer use OCaml-built code for Analysis, but the code should
```

Comment:
I don't follow this comment.

Reply:
The above comment is "Use strings here as js_of_ocaml is not able to deal with int literals outside the 31-bit range"

The Edit: was just an addendum on that comment, noting that while we are no longer using js_of_ocaml, the F# equivalent has assumed the same thing? Maybe I'm wrong.

Same idea for the code comment in Unshared.ml